### PR TITLE
[RFC][NI] Change uni-input-range bindings

### PR DESCRIPTION
**Pull request by AndreJoaquim**
NI

### CODE REVIEW

File: addon/components/uni-input-range.js [hunk 1]
```diff
@@ -13,8 +13,9 @@ export default Ember.Component.extend({
   max: MAX_SAFE_INTEGER,
   min: 0,
   placeholder: "placeholder",
+  onChange: Ember.K,
 
-  isDecrementDisabled: computed('value', function() {
+  isDecrementDisabled: computed('value', function () {
```
Comment (montoias) [inline addon/components/uni-input-range.js]: function() {

File: addon/components/uni-input-range.js [hunk 2]
```diff
@@ -23,19 +24,11 @@ export default Ember.Component.extend({
 
   actions: {
     decrement() {
-      if (this.get('isDecrementDisabled')) {
-        return;
-      }
-
-      this.decrementProperty('value', this.get('offset'));
+      this.get('onChange')(this.get('value') - this.get('offset'));
```
Comment (montoias) [inline addon/components/uni-input-range.js]: use Min here to make sure you don't decrement past the value

File: addon/components/uni-input-range.js [hunk 3]
```diff
@@ -23,19 +24,11 @@ export default Ember.Component.extend({
...
     },
 
     increment() {
-      if (this.get('isIncrementDisabled')) {
-        return;
-      }
-
-      this.incrementProperty('value', this.get('offset'));
+      this.get('onChange')(this.get('value') + this.get('offset'));
```
Comment (montoias) [inline addon/components/uni-input-range.js]: use Max here to make sure you don't decrement past the value

**Review by montoias — CHANGES_REQUESTED** [review]

**Comment by AndreJoaquim** [comment]
@montoias  É carregar no verde

**Pull request merged**